sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -239,6 +239,7 @@ def read_all_items(
     def query_items_change_feed(
         self,
         partition_key_range_id=None,  # type: Optional[str]
+        partition_key=None,  # type: Optional[str]
```

Comment:
This is a breaking change, as these are currently positional parameters.
For example, customers who wrote their code like this are fine:
```python
results = query_items_change_feed(partition_key_range_id='foo', is_start_from_beginning=True)
```
Customers who wrote code like this will be broken:
```python
results = query_items_change_feed('foo', True)
```
The only option to add non-breaking parameters are to add them to end of the method signature or roll them into kwargs. I would suggest the latter in this case, as this method already has too many postitional parameters.

Reply:
updated the PR.